src/components/Card.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { PropsWithChildren } from 'react';
+import { cn } from 'tailwind-cn';
+
+const MessageCard = ({
+  children,
+  senderId
+}: PropsWithChildren<{ senderId: string }>) => {
+  const isSelf = senderId === localStorage.getItem('userId');
```

Comment:
create helper that checks userId not Here also userId may be undefined this means I can get error